XamarinCommunityToolkit.UnitTests/Actions/Snackbar/Snackbar_Tests.cs

```diff
@@ -7,11 +7,13 @@ namespace Xamarin.CommunityToolkit.UnitTests.Actions.Snackbar
 {
 	public class Snackbar_Tests
 	{
+#if !NETCOREAPP
```

Comment:
@VladislavAntonyuk just wonder why did we add it?

Reply:
It executed WPF (.NET Core) implementation of the Snackbar